### PR TITLE
alphabetise authors

### PR DESCRIPTION
**Pull request by callumrollo**
Following recommendation of @vturpin in #148 

# Type of PR

- [x] Typo without possible change of interpretation of the related text.

### CODE REVIEW

**Review by glidermann — APPROVED** [review]

File: OG_Format.adoc [hunk 1]
```diff
@@ -21,16 +21,16 @@ From there the working group organized regular meetings to validate the evolutio
 ////
 == Authors
 
-* https://github.com/castelao[Guilherme Castelão], _Scripps Institution of Oceanography - UC San Diego, USA_
-* https://github.com/vturpin[Victor Turpin], _OceanGliders Technical Coordinator at OceanOPS, Brest, France_
-* https://github.com/kerfoot[John Kerfoot], _Institute of Marine & Coastal Sciences, Rutgers University, New Jersey, USA_
-* https://github.com/soerenthomsen[ Sören Thomsen ], _LOCEAN, ISPL, Sorbonne University, Paris, France,_
+* https://github.com/castelao[Guilherme Castelão] (lead), _Scripps Institution of Oceanography - UC San Diego, USA_
```
Comment (castelao) [inline OG_Format.adoc:24]: I suggest everyone in the leading team be in alphabetical order.

File: OG_Format.adoc [hunk 2]
```diff
@@ -21,16 +21,16 @@ From there the working group organized regular meetings to validate the evolutio
 ////
 == Authors
```
Comment (castelao) [inline OG_Format.adoc:22]: Maybe change to original authors in alphabetical order.

File: OG_Format.adoc [hunk 3]
```diff
@@ -21,16 +21,16 @@ From there the working group organized regular meetings to validate the evolutio
...
-* https://github.com/glidermann[Daniel Hayes], _Cyprus Subsea Consulting and Services C.S.C.S. Ltd, Cyprus_
 * https://github.com/JuangaSocib[Juan Gabriel Fernández Pineda], _SOCIB, Mallorca, Spain_
 * https://github.com/emmerbodc[Emma Gardner], _National Oceanography Centre, British Oceanographic Data Centre, UK_
-* https://github.com/jenseva[Jennifer Sevadjian], _Southwest Fisheries Science Center Environmental Research Division, NOAA, USA_
+* https://github.com/glidermann[Daniel Hayes], _Cyprus Subsea Consulting and Services C.S.C.S. Ltd, Cyprus_
+* https://github.com/kerfoot[John Kerfoot], _Institute of Marine & Coastal Sciences, Rutgers University, New Jersey, USA_
 * https://github.com/callumrollo[Callum Rollo], _Voice of the Ocean Foundation, Sweden_
+* https://github.com/jenseva[Jennifer Sevadjian], _Southwest Fisheries Science Center Environmental Research Division, NOAA, USA_
+* https://github.com/soerenthomsen[ Sören Thomsen ], _LOCEAN, ISPL, Sorbonne University, Paris, France,_
+* https://github.com/vturpin[Victor Turpin], _OceanGliders Technical Coordinator at OceanOPS, Brest, France_
```
Comment (castelao) [inline OG_Format.adoc:34]: Maybe split here adding a subsection:

# Following contributions
* Contributor name, affiliation, ORCID, and contribution brief descriptions.

**Review by castelao — CHANGES_REQUESTED** [review]

**Review by vturpin — APPROVED** [review]

File: OG_Format.adoc (hunk 1 above)
Comment (vturpin) [reply OG_Format.adoc:24]: Agree.

**Review by vturpin — APPROVED** [review]

File: OG_Format.adoc (hunk 2 above)
Comment (vturpin) [reply OG_Format.adoc:22]: Let's do like that. That is fine for me.

**Comment by justinbuck** [comment]
For the ordering of authors, I suspect first too author should the OGDMTT co-chairs as that is where the accountability ultimately lies,, then the rest of us in alphabetical order.

**Comment by jenseva** [comment]
@vturpin - you are generous to include me as an author as I am new to the project and I haven't really contributed much in comparison to others. Feel free to include me last in the list or as an acknowledgement if that is more appropriate! 

My affiliation is the same as Gui's, can you please remove the NOAA SWFSC affiliation and replace it with _Scripps Institution of Oceanography - UC San Diego, USA_?

**Comment by callumrollo** [comment]
Thanks for the input all. To check my understanding we want:
- a two level authors section: lead authors and additional contributors
- sections should be alphabetized
- the lead authors are @castelao @vturpin @glidermann and @justinbuck 

I will make these changes to the PR

@jenseva thanks for the correction. I have applied it in PR #152

**Review by vturpin — APPROVED** [review]

**Review by glidermann — APPROVED** [review]

**Review by vturpin — APPROVED** [review]

**Comment by callumrollo** [comment]
Just needs a review by @castelao then we can merge

**Comment by castelao** [comment]
@callumrollo , thanks for making this move forward. 

Authorship is usually a delicate subject and also important. Beyond the ego, on a practical matter, each one of us has to justify to our own organizations for our time dedicated to this. I appreciate the patience and ethics of everyone, both rare characteristics these days.

I think the suggestion from @justinbuck was to designate it clear who are the co-chairs of OGDMTT, did I get it correct? From the website, it would be @glidermann and @vturpin , is that correct? But I think we share equal accountability, all decisions were voted.

I'm not comfortable with @emmerbodc and @kerfoot in a different group. We've been working together since the start. We organized and attended so many virtual meetings. I think it gives a wrong impression as it is.

What about:

1. Renaming 'project leads' to 'original authors'? And include Emma and John in the original authors.
2. Original authors in alphabetical order.
3. If decided so, Victor and Dan would have a `[OGDMTT co-chair]` after their names.
4. Contributors could be in chronological order, as we expect many more in the future, and this might make it easier to annotate their contributions.
6. We probably should add ORCIDs.
7. This document should have a DOI, and the DOI record should reflect the authors and contributors listed here.

We might be missing more contributors. Maybe I'm mixing the meetings, but I think that there were more people that participated in 1-2 Zoom meetings, or made 1-2 comments in the document. All contributions should count.

An important point that can be postponed is how to think about the future. After the release of OG-1.0, we should probably have a designated chairperson as a contact person. New ideas or requests should be directed to someone to trigger the process. Let's keep this point for another issue/PR to simplify, but probably expect somewhere close to the authors the chairperson(s), a rotating position.

Is everyone comfortable with where your name is?

**Comment by emmerbodc** [comment]
I like @castelao suggestion

**Comment by vturpin** [comment]
I am again happy with the suggestion of @castelao 
'original authors' is probably the best way to put it.
With this organization, we could add back the contributions of Pierre Testor and Thierry Carval I think.

=== Original authors'
* https://github.com/justinbuck[Justin Buck], _National Oceanography Centre, British Oceanographic Data Centre, UK_
* https://github.com/castelao[Guilherme Castelão], _Scripps Institution of Oceanography - UC San Diego, USA_
* https://github.com/emmerbodc[Emma Gardner], _National Oceanography Centre, British Oceanographic Data Centre, UK_
* https://github.com/glidermann[Daniel Hayes], [OGDMTT co-chair], _Cyprus Subsea Consulting and Services C.S.C.S. Ltd, Cyprus_
* https://github.com/kerfoot[John Kerfoot], _Institute of Marine & Coastal Sciences, Rutgers University, New Jersey, USA
* https://github.com/vturpin[Victor Turpin], [OGDMTT co-chair], _OceanGliders Technical Coordinator at OceanOPS, Brest, France_

=== Contributors
* https://github.com/JuangaSocib[Juan Gabriel Fernández Pineda], _SOCIB, Mallorca, Spain_
* https://github.com/callumrollo[Callum Rollo], _Voice of the Ocean Foundation, Sweden_
* https://github.com/jenseva[Jennifer Sevadjian], _Scripps Institution of Oceanography - UC San Diego, USA_
* https://github.com/soerenthomsen[ Sören Thomsen ], _LOCEAN, ISPL, Sorbonne University, Paris, France,_
* https://github.com/pierretestor [Pierre Testor], _LOCEAN, ISPL, Sorbonne University, Paris, France,_
* https://github.com/theirrycarval [Thierry Carval], _IFREMER, Brest, France_

**Comment by callumrollo** [comment]
Thanks for all the recommendations. I've done my best to implement them. I was unable to find orcids for @emmerbodc or @jenseva. Could you post them please? Everyone else, please check yours!

I wasn't present for the early meetings of this format, so if you know people who should be added to the contributors, please add that info in this thread or a PR.

I have used Victor's chronological order of contributors

**Review by vturpin — APPROVED** [review]

**Review by vturpin — APPROVED** [review]

File: OG_Format.adoc [hunk 4]
```diff
@@ -21,16 +21,23 @@ From there the working group organized regular meetings to validate the evolutio
...
-* https://github.com/kerfoot[John Kerfoot], _Institute of Marine & Coastal Sciences, Rutgers University, New Jersey, USA_
-* https://github.com/soerenthomsen[ Sören Thomsen ], _LOCEAN, ISPL, Sorbonne University, Paris, France,_
-* https://github.com/justinbuck[Justin Buck], _National Oceanography Centre, British Oceanographic Data Centre, UK_
-* https://github.com/glidermann[Daniel Hayes], _Cyprus Subsea Consulting and Services C.S.C.S. Ltd, Cyprus_
-* https://github.com/JuangaSocib[Juan Gabriel Fernández Pineda], _SOCIB, Mallorca, Spain_
+
+=== Original authors
+* https://github.com/justinbuck[Justin Buck], _National Oceanography Centre, British Oceanographic Data Centre, UK_ https://orcid.org/0000-0002-3587-4889[0000-0002-3587-4889]
+* https://github.com/castelao[Guilherme Castelão], _Scripps Institution of Oceanography - UC San Diego, USA_ https://orcid.org/0000-0002-6765-0708[0000-0002-6765-0708]
 * https://github.com/emmerbodc[Emma Gardner], _National Oceanography Centre, British Oceanographic Data Centre, UK_
```
Comment (emmerbodc) [inline OG_Format.adoc:28]: @callumrollo  [0009-0007-9640-7268](https://orcid.org/0009-0007-9640-7268)

**Review by emmerbodc — APPROVED** [review]

**Review by vturpin — APPROVED** [review]

**Comment by jenseva** [comment]
Thank you @callumrollo, here is my orcid: https://orcid.org/0000-0001-9286-2043

**Review by castelao — APPROVED** [review]

**Pull request merged**